src/constants/LayoutConstants.js

```diff
@@ -297,4 +297,6 @@ export const DASHBOARD_EDITOR_CARD_TYPES = {
   SIMPLE_BAR: 'SIMPLE_BAR',
   GROUPED_BAR: 'GROUPED_BAR',
   STACKED_BAR: 'STACKED_BAR',
+  LIST: 'LIST',
+  ALERT: 'ALERT',
```

Comment:
I think ALERT shouldn't be added, but instead should be treated as a 'CUSTOM' I didn't think we were going to explicitly support `ALERT` type cards